qemu/tests/cfg/time_conv.cfg

```diff
@@ -0,0 +1,17 @@
+- time_conv:
+    virt_test_type = qemu
+    type = time_conv
+    required_qemu = [5.1.0-2, )
+    start_vm = no
+    force_create_image_stg = yes
+    images = stg
+    image_size_stg = 1G
+    image_name_stg = "images/stg"
+    timeout = 900
+    qemu_img_bench_cmd = "qemu-img bench -f %s -t none -n -w %s -c 100000 -S 8192 -o 0 --pattern=1"
+    qemu_img_conv_cmd = "qemu-img convert -p -f %s -O %s -T none -t none %s %s"
+    raw:
+        max_time = 642.22
+    qcow2:
+        max_time = 598.84
```

Comment:
Only define the threshold for qcow2 and raw, so this case is `only qcow2 raw` ?

Reply:
For luks, I will confirm whether can be supported.

Reply:
It is not only for luck.   If without the default threshold, all of these format need be considered in threory:
https://github.com/avocado-framework/avocado-vt/blob/914dacca6382214aed1ac80ce66d6d5458329ae7/virttest/shared/cfg/guest-hw.cfg#L114

Reply:
I will add "only raw qcow2 luks" in the cfg.